test/integration/connect/envoy/case-api-gateway-http-simple/setup.sh

```diff
@@ -35,6 +35,10 @@ rules = [
     services = [
       {
         name = "s1"
+      },
+      {
+        name = "s2"
+        weight = 2
```

Comment:
We'll need to remove the intentions checks in the verification step (which is fine IMO since they were only used to distinguish between services initially and we do that elsewhere), otherwise this test will fail 2/3 of the time since there's an explicit check for the listener this route binds to of "I can hit this service without an intentions error" and s2 will get intentions errors (and get routed to 2/3 of the time).

Reply:
Updated the test